### ISSUE TITLE
Add sender/receiver address to operation history CSV export

### ISSUE THREAD

**Issue by sap51** [LedgerHQ/ledger-live-desktop#3506]
Hi,

I would like to suggest to be able to export more information when we're using the "export operation history".

In my case, I'm exporting Ledger's data for tax reporting purposes, and I would need to export the Sender and Receiver addresses so I can match them with my addresses that are on other Exchanges/wallets, and also match them with addresses of people I have received the transaction from, or sent the transaction to.

Would it be possible to add a checkbox that allow to user to export the Sender and the Receiver address ?

I know that sometimes there are more than a single address (specially true for the Receiving operations), but that should be fine to separate the multiple addresses with a space in the CSV cell.

Example of checkbox :
![ledger-export](https://user-images.githubusercontent.com/77867893/105566937-815ef300-5d37-11eb-8f53-6c2a54dc333e.jpg)

**Comment by gre** [comment]
The problem is there is not always one sender and one receiver in Bitcoin and altcoins. If you mine/withdraw from an exchange for instance there can be many inputs or many outputs. Indeed most or the time it's always just an address and a change, but it doesn't help on this implementation

**Comment by Th0masL** [comment]
Yes, as I wrote above, sometimes there will be multiple addresses in the Sender/Receiver field, and that's not a problem for me.

In 99% of my cases there will only be one address in each fields, and I'll simply ignore the cells that contains multiples addresses.

In the cases of multiple addresses, they could also be omitted by the exporting tool, I personally don't mind, as my goal is simply to improve the CSV export by not having to type all the addresses manually.

The goal is to let the user decide if he wants to export the addresses or not, hence the check box.

I would have been happy to suggest a PR, but I'm not very good in JavaScript :(

**Comment by sap51** [comment]
Yes, as I wrote above, sometimes there will be multiple addresses in the Sender/Receiver field, and that's not a problem for me.

In 99% of my cases there will only be one address in each fields, and I'll simply ignore the cells that contains multiples addresses.

In the cases of multiple addresses, they could also be omitted by the exporting tool, I personally don't mind, as my goal is simply to improve the CSV export by not having to type all the addresses manually.

The goal is to let the user decide if he wants to export the addresses or not, hence the check box.

I would have been happy to suggest a PR, but I'm not very good in JavaScript :(

**Comment by gre** [comment]
There is probably something we can do indeed. I'll see with the team if we can have a quick win done
